README.md

```diff
@@ -1273,6 +1274,175 @@ Content-Disposition: form-data; name="workflowOptions"; filename="options.json"
 --f3fd038395644de596c460257626edd7--
 ```
 
+## POST /api/workflows/:version/batch
+
+This endpoint accepts a POST request with a `multipart/form-data`
+encoded body.  The form fields that may be included are:
```

Comment:
can you add somewhere the contract that the order of the input jsons will be the order of the returned workflow ids (with better english than I used!)?